### ISSUE TITLE
Touch issues with drive files

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1096]
### Description

On touchscreen devices you have to double press to open a file and long press to open the popup.

### Steps to reproduce

1. open CryptDrive on your touchscreen device
2. try to open a file

### Expected behavior

I should be able to single press to open a file

### What opreating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Google Chrome

### Extensions

none

### Version

5.3.0
